airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2039,6 +2039,155 @@ def execute(self, context) -> None:
         )
 
 
+class BigQueryPatchTableSchemaOperator(BaseOperator):
```

Comment:
`Patch*Operator` are deprecated call `BigQueryUpdateTableSchemaOperator` to keep actual convention.

Reply:
changed